### ISSUE TITLE
Wallet: share acount bottom sheet action

### ISSUE THREAD

**Issue by J-Son89** [status-im/status-react#17766]
on this bottom sheet - https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?type=design&node-id=2338-508395&mode=design&t=K0hD3hXfHyjRCciH-4

and this bottom sheet there is the action "share account" ->
implement this action that follow these as per designs. 

As currently understood it will use native share features however please align with design before starting

**Comment by OmarBasem** [comment]
@J-Son89 is this still blocked?

**Comment by J-Son89** [comment]
Yeah it's still blocked, we don't have designs for share account -

and share address we should wait for ->
https://github.com/status-im/status-mobile/issues/17865

because share address in wallet bottom sheets will open this on the particular address/account

**Comment by J-Son89** [comment]
@OmarBasem - I believe this is not blocked. 
We can however focus on share account feature and I will create a separate issue to work on share address from this bottom sheet.
As far as I understand it the share account feature will use the native ios share feature. 
Please sync with design team before working on this. 👍

**Comment by OmarBasem** [comment]
@J-Son89 is this issue for sharing the QR or sharing the address via the iOS native share?
